addon/components/file-picker.js

```diff
@@ -211,39 +211,47 @@ export default Component.extend({
   },
   /* Drag'n'Drop events */
   dragOver: function(event) {
-    if (event.preventDefault) {
-      event.preventDefault();
+    if (this.get('dropzone') === true) {
+      if (event.preventDefault) {
```

Comment:
What do we accomplish by calling `preventDefault` when we're not handling a dnd operation?